res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -0,0 +1,2407 @@
+////////////////////////////////////////////////////////////////////////
+// JSHint configuration                                               //
+////////////////////////////////////////////////////////////////////////
+/* global engine                                                      */
+/* global script                                                      */
+/* global midi                                                        */
+/* global bpm                                                         */
+////////////////////////////////////////////////////////////////////////
+var PioneerDDJSX = function() {};
+
+/*
+	Author: 		DJMaxergy
+	Version: 		1.09, 05/14/2017
+	Description: 	Pioneer DDJ-SX Controller Mapping for Mixxx
+    Source: 		http://github.com/DJMaxergy/mixxx/tree/pioneerDDJSX_mapping
+    
+    This mapping for the Pioneer DDJ-SX was made by DJMaxergy, Maximilian Beiersdorfer.
+    Basing on DDJ-SB2 for Mixxx 2.0 by Michael Stahl (https://github.com/dg3nec/mixxx/tree/DDJ-SB2/res/controllers),
+    basing on midiAutoDJ-scripts by Sophia Herzog,
+    basing on the work of wingcom (wwingcomm@gmail.com, https://github.com/wingcom/Mixxx-Pioneer-DDJ-SB),
+    which in turn was based on the work of Hilton Rudham (https://github.com/hrudham/Mixxx-Pioneer-DDJ-SR).
+    Just as wingcom's and Rudham's work, PioneerDDJSX mapping is pusblished under the MIT license.
+    
+*/
+
+///////////////////////////////////////////////////////////////
+//                       USER OPTIONS                        //
+///////////////////////////////////////////////////////////////
+
+// Sets the jogwheels sensivity. 1 is default, 2 is twice as sensitive, 0.5 is half as sensitive.
+PioneerDDJSX.jogwheelSensivity = 1;
+
+// Sets how much more sensitive the jogwheels get when holding shift.
+// Set to 1 to disable jogwheel sensitivity increase when holding shift (default: 10).
+PioneerDDJSX.jogwheelShiftMultiplier = 10;
+
+// Sets the default speed slider range (default: 0.08 = 8%).
+PioneerDDJSX.speedSliderRange = 0.08;
```

Comment:
I don't think this should be here. The logic for changing the rate slider range with shift + keylock should work regardless of what value it starts at.